WORKSPACE

```diff
@@ -0,0 +1 @@
+workspace(name = "nlohmann_json")
```

Comment:
Is this file really needed in this repository?

Reply:
Yes it is - otherwise, this is not recognized by Bazel - I named the file `WORKSPCE.bazel` (which is an alternative to WORKSPACE only) to make it more clear that this is a Bazel thing